docs/FAQ/multi_devices.md

```diff
@@ -148,6 +148,13 @@ Note that here we
 - `-n` number of worker nodes to run on
 - `-H` the host file which is required by `ssh` and `mpi`
 - `--kv-store` use either `dist_sync` or `dist_async`
+- If the `-s` argument is not passed, it will keep the number of servers same as number of workers
```

Comment:
Should we first mention what `-s` is for?

Reply:
Added!